render/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/render/jpql/serializer/impl/JpqlLocalDateTimeSerializer.kt

```diff
@@ -0,0 +1,27 @@
+package com.linecorp.kotlinjdsl.render.jpql.serializer.impl
+
+import com.linecorp.kotlinjdsl.Internal
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.impl.JpqlLocalDateTime
+import com.linecorp.kotlinjdsl.render.RenderContext
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlRenderSerializer
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlSerializer
+import com.linecorp.kotlinjdsl.render.jpql.writer.JpqlWriter
+import kotlin.reflect.KClass
+
+@Internal
+class JpqlLocalDateTimeSerializer : JpqlSerializer<JpqlLocalDateTime> {
+
+    override fun handledType(): KClass<JpqlLocalDateTime> {
+        return JpqlLocalDateTime::class
+    }
+
+    override fun serialize(part: JpqlLocalDateTime, writer: JpqlWriter, context: RenderContext) {
+        val delegate = context.getValue(JpqlRenderSerializer)
+
+        writer.write("LOCAL DATETIME")
+
+        writer.writeParentheses {
+            delegate.serialize(part, writer, context)
+        }
```

Comment:
I checked that `LOCAL DATETIME` does not need parentheses. Please remove the parentheses.